netsim/install/libvirt.sh

```diff
@@ -50,6 +50,13 @@ set -e
 # changed to new method - ghostinthenet - 20220417
 curl -fsSL https://apt.releases.hashicorp.com/gpg | $SUDO gpg --dearmor -o /etc/apt/trusted.gpg.d/hashicorp-security.gpg
 $SUDO sh -c 'echo "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" > /etc/apt/sources.list.d/vagrant.list'
+#
+# Pin vagrant version to one we know works
+cat <<FILE | $SUDO tee /etc/apt/preferences.d/vagrant
+Package: vagrant
+Pin: version 2.4.3-1
```

Comment:
Can we add a comment that this is being pinned by Netlab install script, with explanation that later versions were found to have issues (on 6/25/2025)

Reply:
Sure, makes perfect sense. Thanks a million for the suggestion!